ovn_cluster.sh

```diff
@@ -592,95 +679,118 @@ function start() {
-ovn-nbctl lsp-set-addresses public-lr0 router
-ovn-nbctl lsp-set-options public-lr0 router-port=lr0-public
+ovn-nbctl lr-add lr\$az
+ovn-nbctl lrp-add lr\$az lr\$az-sw0\$az 00:0\$az:00:00:ff:01 1\$az.0.0.1/24 100\$az::a/64
+ovn-nbctl lsp-add sw0\$az sw0\$az-lr\$az
+ovn-nbctl lsp-set-type sw0\$az-lr\$az router
+ovn-nbctl lsp-set-addresses sw0\$az-lr\$az router
+ovn-nbctl lsp-set-options sw0\$az-lr\$az router-port=lr\$az-sw0\$az
+
+ovn-nbctl lrp-add lr\$az lr\$az-sw1\$az 00:0\$az:00:00:ff:02 2\$az.0.0.1/24 200\$az::a/64
+ovn-nbctl lsp-add sw1\$az sw1\$az-lr\$az
+ovn-nbctl lsp-set-type sw1\$az-lr\$az router
+ovn-nbctl lsp-set-addresses sw1\$az-lr\$az router
+ovn-nbctl lsp-set-options sw1\$az-lr\$az router-port=lr\$az-sw1\$az
+
+ovn-nbctl ls-add public\$az
+ovn-nbctl lrp-add lr\$az lr\$az-public\$az 00:0\$az:20:20:12:13 172.16.\$az.100/24 300\$az::a/64
+ovn-nbctl lsp-add public\$az public\$az-lr\$az
+ovn-nbctl lsp-set-type public\$az-lr\$az router
+ovn-nbctl lsp-set-addresses public\$az-lr\$az router
+ovn-nbctl lsp-set-options public\$az-lr\$az router-port=lr\$az-public\$az
 
 # localnet port
-ovn-nbctl lsp-add public ln-public
-ovn-nbctl lsp-set-type ln-public localnet
-ovn-nbctl lsp-set-addresses ln-public unknown
-ovn-nbctl lsp-set-options ln-public network_name=public
+ovn-nbctl lsp-add public\$az ln-public\$az
+ovn-nbctl lsp-set-type ln-public\$az localnet
+ovn-nbctl lsp-set-addresses ln-public\$az unknown
+ovn-nbctl lsp-set-options ln-public\$az network_name=public
 
 # schedule the gw router port to a chassis.
-ovn-nbctl lrp-set-gateway-chassis lr0-public ovn-gw-1 20
+ovn-nbctl lrp-set-gateway-chassis lr\$az-public\$az ovn-gw-\$az 20
 
 # Create NAT entries for the ports
 
-# sw0-port1
-ovn-nbctl lr-nat-add lr0 dnat_and_snat 172.16.0.110 10.0.0.3 sw0-port1 30:54:00:00:00:03
-ovn-nbctl lr-nat-add lr0 dnat_and_snat 3000::c 1000::3 sw0-port1 40:54:00:00:00:03
-# sw1-port1
-ovn-nbctl lr-nat-add lr0 dnat_and_snat 172.16.0.120 20.0.0.3 sw1-port1 30:54:00:00:00:04
-ovn-nbctl lr-nat-add lr0 dnat_and_snat 3000::d 2000::3 sw1-port1 40:54:00:00:00:04
+# sw0\$az-port1
+ovn-nbctl lr-nat-add lr\$az dnat_and_snat 172.16.\$az.110 1\$az.0.0.3 sw0\$az-port1 30:5\$az:00:00:00:03
+ovn-nbctl lr-nat-add lr\$az dnat_and_snat 300\$az::c 100\$az::3 sw0\$az-port1 40:5\$az:00:00:00:03
+# sw1\$az-port1
+ovn-nbctl lr-nat-add lr\$az dnat_and_snat 172.16.\$az.120 2\$az.0.0.3 sw1\$az-port1 30:5\$az:00:00:00:04
+ovn-nbctl lr-nat-add lr\$az dnat_and_snat 300\$az::d 200\$az::3 sw1\$az-port1 40:5\$az:00:00:00:04
 
 # Add a snat entry
-ovn-nbctl lr-nat-add lr0 snat 172.16.0.100 10.0.0.0/24
-ovn-nbctl lr-nat-add lr0 snat 172.16.0.100 20.0.0.0/24
+ovn-nbctl lr-nat-add lr\$az snat 172.16.\$az.100 1\$az.0.0.0/24
+ovn-nbctl lr-nat-add lr\$az snat 172.16.\$az.100 2\$az.0.0.0/24
+
+if [ "\$ic" == "yes" ]; then
+    ovn-nbctl set NB_Global . name=az\$az
+    ovn-nbctl set NB_Global . options:ic-route-adv=true options:ic-route-learn=true
+
+    if [ \$az -eq 1 ]; then
```

Comment:
This part needs to happen regardless of the CREATE_FAKE_VMS value.  We want a working (even if empty) multi-AZ cluster.  This incremental change should deal with it, wdyt?
```
diff --git a/ovn_cluster.sh b/ovn_cluster.sh
index eb6ab48..b519881 100755
--- a/ovn_cluster.sh
+++ b/ovn_cluster.sh
@@ -636,10 +636,19 @@ function start() {
             ${RUNC_CMD} exec ${central} ovn-nbctl set-connection p${REMOTE_PROT}:6641
             ${RUNC_CMD} exec ${central} ovn-nbctl set connection . inactivity_probe=180000
 
+            ${RUNC_CMD} exec ${central} ovn-nbctl set NB_Global . name=${central} \
+                options:ic-route-adv=true options:ic-route-learn=true
+
             ${RUNC_CMD} exec ${central} ovn-sbctl set-connection p${REMOTE_PROT}:6642
             ${RUNC_CMD} exec ${central} ovn-sbctl set connection . inactivity_probe=180000
         done
 
+        ${RUNC_CMD} exec ${CENTRAL_NAMES[0]} ovn-ic-nbctl ts-add ts1
+        # wait for ovn-ic to kick in
+        while sleep 2; do
+            ${RUNC_CMD} exec ${CENTRAL_NAMES[0]} ovn-nbctl ls-list | grep -q ts1 && break
+        done
+
         for name in "${RELAY_NAMES[@]}"; do
             SSL_ARGS=
             if [ "$ENABLE_SSL" == "yes" ]; then
@@ -766,18 +775,6 @@ ovn-nbctl lr-nat-add lr\$az snat 172.16.\$az.100 1\$az.0.0.0/24
 ovn-nbctl lr-nat-add lr\$az snat 172.16.\$az.100 2\$az.0.0.0/24
 
 if [ "\$ic" == "yes" ]; then
-    ovn-nbctl set NB_Global . name=az\$az
-    ovn-nbctl set NB_Global . options:ic-route-adv=true options:ic-route-learn=true
-
-    if [ \$az -eq 1 ]; then
-        ovn-ic-nbctl ts-add ts1
-
-        # wait for ovn-ic to kick in
-        while sleep 2; do
-            ovn-nbctl ls-list | grep -q ts1
-            [ \$? -eq 0 ] && break
-        done
-    fi
     ovn-nbctl lrp-add lr\$az lr\$az-ts1 aa:aa:aa:aa:aa:0\$az 5.0.0.\$az/24
     ovn-nbctl lsp-add ts1 ts1-lr\$az -- lsp-set-addresses ts1-lr\$az -- lsp-set-type ts1-lr\$az router -- lsp-set-options ts1-lr\$az  router-port=lr\$az-ts1
     ovn-nbctl lrp-set-gateway-chassis lr\$az-ts1 ovn-gw-\$az 1
```

Reply:
@dceara ack, I will fix it